MEGA/Main/Main.ino

```diff
@@ -121,17 +121,88 @@ State state();
 #endif
 
 #if ENABLE_SENSORS
+#define RGB_1_COLOUROUT A5
+#define RGB_1_S2 A3
+#define RGB_1_S3 A4
+#define RGB_2_COLOUROUT A10
+#define RGB_2_S2 A8
+#define RGB_2_S3 A9 
+
   class Sensors {
+    /* Sensor Type : RGB sensors 
+       Sensor Model : XC3708 
+       Number of sensors : 2 (Front and Rear of the carriage) */
+
+    private: 
+      int baudRate = 9600;
+
     public:
+    
       Sensors() {
         // Use ::state to access the state class in this scope
+        Serial.begin(baudRate);
```

Comment:
We'll most likely have to call this in the setup function rather than in the constructors of each subsystem class.

Reply:
Perfect will fix that !